src/R.Settings.targets

```diff
@@ -3,10 +3,10 @@
   <!-- Common Build Settings and Configurations -->
   <PropertyGroup>
     <DefineConstants Condition="'$(VisualStudioVersion)'=='14.0'">$(DefineConstants);VS14</DefineConstants>
-    <RootDirectory Condition="'$(RootDirectory)' == ''">$(MSBuildThisFileDirectory)\..\</RootDirectory>
-    <LibDirectory Condition="'$(LibDirectory)' == ''">$(RootDirectory)\lib\</LibDirectory>
-    <BinDirectory Condition="'$(BinDirectory)' == ''">$(RootDirectory)\bin\</BinDirectory>
-    <ObjDirectory Condition="'$(ObjDirectory)' == ''">$(RootDirectory)\obj\</ObjDirectory>
+    <RootDirectory Condition="'$(RootDirectory)' == ''">$(MSBuildThisFileDirectory)..</RootDirectory>
```

Comment:
maybe, but all built-in variables include trailing backslashes, so I'd like to be consistent with them.

---
In reply to: [60325204](https://github.com/Microsoft/RTVS/pull/1648#discussion_r60325204) [](ancestors = 60325204)